### DISCUSSION TITLE
Agent installation Documentation

### DISCUSSION THREAD

**Discussion by MagoMaverick** [immense/Remotely#530] [Q&A] [answered]
Hi,
I Installed the server version on our Windows 2019 server, and service is online.

Now I want to add an agent to have unattended remote control of that machine. 
So I put the agent and try to start but it ask for a ConnectionInfo.json
I create a json file like this:
{
  "organization":"My Organization",
  "host":"https://remote.myorganization.it:myport",
  "device":"name of server"
}

myorganization is an example and myport is a number opened through firewall that we use to connect using https to Remotely service.

Anyway I get this message:

An invalid request URI was provided. Either the request URI must be an absolute URI or BaseAddress must be set.

I tried without https://, without port without success.

There is some documentation about launching Agent correctly?

Thank you in advance.

**Comment by xboxscanner** [comment, marked as answer]
I don't think there is any docs on this,
 I have just installed it and have to try it, by trial and error.

open the downloads menu and download the installer "windows installer(x64/x86) this will download a file called "Remotely_Install-[xxxx].exe". The xxxx is a relay code to identify your server, I tried this without the code and it installed but did not appear on the home dashboard. 
I also used the silent install command (first example) through powershell removing the -silent switch, only so that i can see that it does install. 
you will also need to access the portal from a different machine or else you will get an infinity loop, much like the infinity mirror effect when there are two mirrors facing each other.

**Reply by MagoMaverick** [reply]
Hi, Thank you, I Just found in the Download page all the info I needed. I was trying to publish the agent without installer using VS.
Now I'm done!
